File: daraja/src/main/java/com/androidstudy/daraja/model/ErrorResponse.java
package com.androidstudy.daraja.model;

import com.google.gson.annotations.SerializedName;

public class ErrorResponse {

    @SerializedName("requestId")
    private String requestId;

    @SerializedName("errorCode")
    private String code;

    @SerializedName("errorMessage")
    private String message;

    public String getRequestId() {
        return requestId;
    }

    public void setRequestId(String requestId) {
        this.requestId = requestId;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: daraja/src/main/java/com/androidstudy/daraja/model/ErrorResponse.kt
package com.androidstudy.daraja.model

import com.google.gson.annotations.SerializedName

data class ErrorResponse(
        @SerializedName("requestId")
        var requestId: String,
        @SerializedName("errorCode")
        var code: String,
        @SerializedName("errorMessage")
        var message: String
)